lib/helpers/log.js

```diff
@@ -2,32 +2,35 @@
 
 require('colors');
 
-var debugLog = function() {};
+const noOp = () => {};
 
-if (process.env.OBT_DEBUG === 'true') {
-	debugLog = function(text) {
-		console.log(String(text).grey);
-	};
-}
+const logLevelNormal = process.env.OBT_LOG_LEVEL !== 'none';
+const logLevelDebug = process.env.OBT_LOG_LEVEL === 'debug';
```

Comment:
I was a tad confused by this naming, maybe use `minLogLevel`.  The current names initially sound like constants for bitwise flags.  Not a big thing.